src/Sdk/NewsroomContacts/NewsroomContacts.ts

```diff
@@ -11,6 +11,7 @@ import {
 } from './types';
 
 type NewsroomId = Newsroom['uuid'] | Newsroom['id'];
+type ContactId = NewsroomContact['uuid'] | NewsroomContact['id'];
```

Comment:
```suggestion
type NewsroomContactId = NewsroomContact['uuid'] | NewsroomContact['id'];
```

`NewsroomContact` is significantly different from a `Contact`, let's not introduce any confusion.

Reply:
37fbbc3